src/nft/components/bag/BagFooter.tsx

```diff
@@ -71,17 +77,26 @@ const WarningText = styled(ThemedText.BodyPrimary)`
   color: ${({ theme }) => theme.accentWarning};
   display: flex;
   justify-content: center;
-  margin: 12px 0 !important;
+  margin-bottom: 10px !important;
```

Comment:
nit: I don't think the `!important` is needed

Reply:
surprisingly it is